update-version.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+version=$(jq -r '.version' package.json)
+jq --arg version $version '.version |= $version' version.json >file.json.tmp && cp file.json.tmp version.json
```

Comment:
question: Why do we need to write to `file.json.tmp` first?

Reply:
If we write directly to the `version.json` file, it appends onto the content instead of overwriting the file so we have to write to a temp file and then copy the file over to overwrite the existing file.

Reply:
Given we're ignoring the version.json file and removed file from changes, I updated the script so that we just write to a new file.